### PR TITLE
Add tests for correct behavior of :vars

### PR DESCRIPTION
**Pull request by schillic**

### CODE REVIEW

File: test/Reachability/solve_continuous.jl [hunk 1]
```diff
@@ -15,17 +15,21 @@ A = [ 0.0509836  0.168159  0.95246   0.33644
 X0 = BallInf(ones(4), 0.1)
 
 # default options (computes all variables)
-s = solve(InitialValueProblem(LinearContinuousSystem(A), X0), :T=>0.1)
+s = solve(InitialValueProblem(LinearContinuousSystem(A), X0),
```
Comment (mforets) [inline test/Reachability/solve_continuous.jl]: i would prefer to keep the "default options" test and add the test where you specifically give `project_reachset` separately, what do you think?

Comment (schillic) [reply test/Reachability/solve_continuous.jl]: I can remove the option after merging #486 again.

**Comment by schillic** [comment]
No idea what the build complains about. I just tried again in current `master` and the tests pass.

**Comment by mforets** [comment]
let's merge it

**Pull request merged**